Source/Core/DiscIO/VolumeWiiCrypted.h

```diff
@@ -56,11 +56,12 @@ class CVolumeWiiCrypted : public IVolume
   static const unsigned int s_block_data_size = 0x7C00;
   static const unsigned int s_block_total_size = s_block_header_size + s_block_data_size;
 
+  static const u64 s_partition_data_offset = 0x20000;
```

Comment:
Please don't use the s_ prefix for constants. Looking around the rest of dolphin's code base, we often use no prefix for class constants, or `NO_PREFIX_ALL_CAPS`. Can you change the existing constants too?

Also, these should possibly be constexpr, @lioncash is the expert on this issue.